src/main/java/com/amazon/kinesis/kafka/AmazonKinesisSinkTask.java

```diff
@@ -240,6 +251,16 @@ public void start(Map<String, String> props) {
 
 		regionName = props.get(AmazonKinesisSinkConnector.REGION);
 
+		roleARN = props.get(AmazonKinesisSinkConnector.ROLE_ARN);
+
+		roleSessionName = props.get(AmazonKinesisSinkConnector.ROLE_SESSION_NAME);
+
+		roleDurationSeconds =  Integer.parseInt(props.get(AmazonKinesisSinkConnector.ROLE_DURATION_SECONDS));
```

Comment:
In readme, this attribute is 'optional'. If not provided, in properties, would this fail on parsing?

Reply:
A default value of 3600 seconds is used if no value specified: https://github.com/awslabs/kinesis-kafka-connector/compare/master...skidder:sk-add-role-and-external-id-support#diff-49b5e2a800087a94262719a073393d45R118-R121